spec/labor/rate_limit_checker_spec.rb

```diff
@@ -4,6 +4,17 @@
   let(:user) { create(:user) }
   let(:article) { create(:article, user_id: user.id) }
 
+  describe "self.daily_account_follow_limit " do
+    it "returns 500 by default" do
+      expect(described_class.daily_account_follow_limit).to eq(500)
+    end
+
+    it 'returns the value set in ENV["RATE_LIMIT_FOLLOW_COUNT_DAILY"] as an integer if it\'s set' do
+      ENV["RATE_LIMIT_FOLLOW_COUNT_DAILY"] = "20"
```

Comment:
This is potentially dangerous, because you're modified the environment variable for ALL tests. 
You should just check that `described_class.daily_account_follow_limit` is equal to `ApplicationConfig["RATE_LIMIT_FOLLOW_COUNT_DAILY"]`